docs/docs/misc/migration_notes.md

```diff
@@ -6,6 +6,23 @@ keywords: [sandbox, cli, aztec, notes, migration, updating, upgrading]
 
 Aztec is in full-speed development. Literally every version breaks compatibility with the previous ones. This page attempts to target errors and difficulties you might encounter when upgrading, and how to resolve them.
 
+## TBD
+
+### [Aztec.nr] Keys: Token note now stores an owner master nullifying public key hash instead of an owner address.
+
+i.e.
+
+struct TokenNote {
+    amount: U128,
+    ```diff
```

Comment:
As Lasse pointed out, docs should change in several places here (such as token tutorial where we add address or anywhere where we talk about notes! This can also be done in your other PR where you are changing all other notes